TicTacToe/TicTacToe/Controllers/SettingsPageViewController.swift

```diff
@@ -0,0 +1,160 @@
+            footerLabel.trailingAnchor.constraint(equalTo: footerContainerView.trailingAnchor, constant: -16),
+            footerLabel.bottomAnchor.constraint(equalTo: footerContainerView.bottomAnchor),
+        ])
+        settingsPageView.tableView.tableFooterView = footerContainerView
+    }
+}
+
+extension SettingsPageViewController: UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 3
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case 0:
+            return 1
+        case 1:
+            return 1
+        case 2:
+            return 4
+        default:
+            break
+        }
+        return 0
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch indexPath.section {
+        case 0:
+            let cell = settingsPageView.tableView.dequeueReusableCell(withIdentifier: textWithIconCellIdentifier, for: indexPath) as! TextWithIconCell
+            cell.configure(text: "About")
+            return cell
+        case 1:
+            let cell = settingsPageView.tableView.dequeueReusableCell(withIdentifier: textWithIconCellIdentifier, for: indexPath) as! TextWithIconCell
+            cell.configure(text: "Rules")
+            return cell
+        case 2:
+            let cell = settingsPageView.tableView.dequeueReusableCell(withIdentifier: textWithSecondaryTextCellIdentifier, for: indexPath) as! TextWithSecondaryTextCell
+            let text: String
+            let secondaryText: String
+            switch indexPath.row {
+            case 0:
+                text = "Level"
+                secondaryText = Settings.shared.level.rawValue
+            case 1:
+                text = "Opponent"
+                secondaryText = Settings.shared.opponent.rawValue
+            case 2:
+                text = "First Turn"
+                secondaryText = Settings.shared.firstTurn.rawValue
+            case 3:
+                text = "My Symbol"
+                secondaryText = Settings.shared.mySymbol.rawValue
+            default:
+                return UITableViewCell()
+            }
+            cell.configure(text: text, secondaryText: secondaryText)
+            return cell
+        default:
+            return UITableViewCell()
+        }
+        return UITableViewCell()
+    }
+    
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        if indexPath.row == tableView.numberOfRows(inSection: indexPath.section) - 1 {
+            addTableFooterView()
+        }
+    }
+}
+
+extension SettingsPageViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        if (indexPath.section == 2) {
```

Comment:
No need of braces in if check.
```suggestion
        if indexPath.section == 2 {
```